_release_notes/armoryspinnaker_v1.14.65.md

```diff
@@ -0,0 +1,926 @@
+
+
+
+
+
+<br><br><br>
+## Detailed Updates
+### Armory
+### Lighthouse&trade; - d7395c7...e70f1e6
+ - upgrade path: Strip quotes from parsing the annotations file. (#218)
+ - upgrade path: SETTINGS is a FlatDict, not a dict; need to coerce. (#217)
+ - upgrade path: ENG-2126 override upgrade manifest (#216)
+ - k8s installation: ENG-2115 postinstall notes (#215)
+ - k8s installation: fix(health) OSS health checks are on /health (#214)
+ - k8s installation: Permitting the use of a "wildcard" migration. (#212)
+ - k8s installation: ENG-2083: Update armoryVersion in pipeline after migration (#211)
+ - k8s installation: Remove the migration_paths_cache. (#210)
+ - k8s installation: Add a /v1/upgrade/current endpoint (#209)
+ - k8s installation: Allow a migration definition to be a dict (#207)
+ - k8s installation: ENG-1963: add routes for front50 (#206)
+ - k8s installation: ENG-1955 add config migrator (#205)
+
+### Dinghy&trade; - 2015849...f8a6984
+ - Add download cache for Github and Stash (#65)
+ - feat(github): support github enterprise endpoints (#64)
+ - feat(config): merge defualt config with -local (#63)
+ - Revert "feat(github): make github endpoint configurable (#61)" (#62)
+ - feat(github): make github endpoint configurable (#61)
+ - option to log to file (#60)
+ - close http response body the right way
+ - The default value of a var can now come from another var. That var has to be prefixed with an @. e.g.,: "{{ var "discovery-service-name" ?: "@application" }}"
+
+### Platform&trade; - ff5e90c...8812305
+ - enable NR APM instrumentation (#204)
+ - feat(logging) log each time we're making a request to slack
+ - Removed GitHubOrg from AppCreationTask struct and put it in the template since it can potentially be a different org per template, also removed hardcoding of the org and fixed some typos in env var names (#201)
+ - feat(logging) add more debug logging (#200)
+ - store app creation status in redis (#199)
+ - feat(git) ignore vendor.orig (#193)
+ - feat(logging) improve debug logging (#196)
+ - fix(slack) if payload isn't expected, we should return an actual error (#195)
+ - feat(slack) slack message improvements (#194)
+ - feat(readme) add slackbot icon to readme
+ - feat(assets) add armorybot slack icons
+ - feat(settings) try to fetch the password from yaml (#197)
+ - read namespace from config (#192)
+ - change namespace to K8s account available in dev. This should be re-worked so that we read the namespace either from settings or from the UI (#190)
+ - fix(rollback) orca changed it's endpoint and content type for tasks (#189)
+
+### Armory Echo  - 00219be...e8f6375
+ - refactor(jira) rename Armory Jira classes due to OSS conflict (#64)
+ - If the substitutions are null, replace with blank. (#63)
+ - feat(tests) add doc on how which integration tests to run (#62)
+ - fix(jira) make sure to check all tickets (#61)
+ - feat(jira) support multiple scms (#60)
+ - Add README notes (#58)
```

Comment:
Probably remove #58, #62, #63, #64 -- these are all internal changes that are meaningless to customers?